3.10/release-notes-new-features310.md

```diff
@@ -29,7 +29,30 @@ no move operations have been scheduled if they are not necessary.
 AQL
 ---
 
+### GeoJSON changes
```

Comment:
This is the controversial point. As @neunhoef mentioned, we need to carefully craft this message. Perhaps spinning it around.

"The 3.10 release of ArangoDB conforms to the standards specified in GeoJSON objects... and models...
This diverges from the previous implementation in two fundamental ways:
1.   ...
2. ....

For existing users that do not wish to rebuild their indices and continue using the previous behaviour, a flag 'legacyPolygon' [...] has been introduced to guarantee backwards compatibility.
For existing users that wish to take advantage of the new standard behaviour,, geo indexes need to be dropped and.... [...]

See .[...] upgrading to version 3.10 or later.
....
Additionally, the reported issues that where occasionally producing wrong results in geo queries when using geo indexes have been fixed. " (and perhaps link to the detail of what's been fixed?)

Reply:
Changed.